### PR TITLE
Modules - Stringtabled finally!

### PR DESCRIPTION
**Pull request by gienkov**
I made mess so new pull request.

Done:

* Advanced Ballistics module
* Ballistics - ammo crate
* Captives module
* Common module
* 2 Vehiclelock modules
* Weather module
* Winddeflection module
* Hearing module
* Interaction module
* Map and BFT modules
* SwitchUnits module
* Ambiance Sounds - module, category
* Medical - all 6 modules, 2 chests, editor category, fixes to CMS leftovers in descriptions
* MK6 Settings - module
* Name Tags - module
* Rallypoint System - module + move rallypoint action
* Respawn System - module + editor category
* Friendly Fire Messages - module
* Explosive System - module
* Allow Config Export [ACE] - module
* MicroDAGR Map Fill - module

Also:
* Added module descriptions to all modules (where they were missing)
* Fixed #976 
* Missing translations inside options menu and that were: missileguidance, common, nametags

Left:
* Nothing, I kindly ask for a merge

### CODE REVIEW

**Comment by gienkov** [comment]
This is ready for merge now. I kindly ask to show my nick inside medical pbo since that required almost 2 hours to complete. All work was about 8 hours or so (with testing if everything works).
Thanks.

**Comment by gienkov** [comment]
This is done now. It taken me about 8 hours or more to do since I had to learn how to make this work and Im not that good at scripting. It was easy to do anyway but very boring from start to the end. Since I like to have everything in my native language I forced myself to do all of this for you guys.
Also, I kindly ask to have my nick inside authors of medical - translating that taken most of my time and I also fixed one string that was not translated into stringtables.
It would be nice to have this inside first official release version ;).

File: addons/hearing/CfgVehicles.hpp [hunk 1]
```diff
@@ -98,21 +98,24 @@ class CfgVehicles {
...
             class EnableCombatDeafness {
-                displayName = "Enable combat deafness?";
-                description = "Enable combat deafness?";
+                displayName = "$STR_ACE_Hearing_CombatDeafness_DisplayName"; //Enable combat deafness?
+                description = "$STR_ACE_Hearing_CombatDeafness_Description"; //Enable combat deafness?
                 typeName = "BOOL";
                 class values {
-                    class Yes { name = "Yes"; value = 1; default = 1; };
-                    class No { name = "No"; value = 0; };
+                    class Yes { name = "$STR_ACE_Hearing_CombatDeafness_Yes"; value = 1; default = 1; }; //yes
```
Comment (bux578) [inline addons/hearing/CfgVehicles.hpp]: This shouldn't be necessary. `Yes` and `No` is already translated in vanilla. And we should change the code here to how @glowbal did it in medical: https://github.com/acemod/ACE3/blob/master/addons/medical/CfgVehicles.hpp#L60-L65

```
typeName = "BOOL";
defaultValue = 1;
```

**Comment by bux578** [comment]
Thank you @gienkov!

Next time **please please please** use better commit messages.

```
bla
bbb
next
```

are not very descriptive.

**Comment by gienkov** [comment]
Sorry about that, tought it was not that much important :).

2015-05-04 8:54 GMT+02:00 bux578 <notifications@github.com>:

**Comment by KoffeinFlummi** [comment]
Relevant XKCD: https://xkcd.com/1296/

But yeah, this is a clear case for some rebasing... :sunglasses:

**Comment by esteldunedain** [comment]
> Since I like to have everything in my native language I forced myself to do all of this for you guys.

Good for you; but that sentence is a bit contradictory.

> Also, I kindly ask to have my nick inside authors of medical - translating that taken most of my time and I also fixed one string that was not translated into stringtables.

I'd say that time is **nowhere** near to that spent by the module authors. So imo that's still a clear no.

Thanks for the contribution though.

**Comment by gienkov** [comment]
Ok then. Glad I could help anyway.
4 maj 2015 14:17 "Nicolás Badano" <notifications@github.com> napisał(a):
>>
>> Since I like to have everything in my native language I forced myself to
do all of this for you guys.
>
> Good for you; but that sentence is a bit contradictory.
>>
>> Also, I kindly ask to have my nick inside authors of medical -
translating that taken most of my time and I also fixed one string that was
not translated into stringtables.
>
> I'd say that time is nowhere near to that spent by the module authors. So
imo that's still a clear no.

**Comment by alganthe** [comment]
oh rebasing is needed here, that or you make it up to 100 commits :smile:

**Comment by gienkov** [comment]
More elegant pull request #1084

**Pull request closed without merging**